### DISCUSSION TITLE
Comparison of justpy vs. dash

### DISCUSSION THREAD

**Discussion by bigfatwhale** [elimintz/justpy#615] [General]
Hi, i've used mostly justpy in my work and recently came across dash. Anyone with more familiarity with dash who can discuss the pros-and-cons of the two? From a cursory look it seems like justpy operates at a "lower-level" than dash, would that be an accurate description?

**Comment by WolfgangFahl** [comment]
As an answer to https://www.reddit.com/r/Python/comments/ybe1z4/justpy_open_source_community_is_growing/ there was a pointer to https://may69.com/purepython in which dash is rated "Class B" together with justpy. Maybe you'll find some criteria you find interesting in that list.
